pkg/pulumiyaml/codegen/load.go

```diff
@@ -296,6 +317,8 @@ func (imp *importer) importBuiltin(node ast.BuiltinExpr) (model.Expression, synt
 			Collection: values,
 			Key:        index,
 		}, diags
+	case *ast.SplitExpr:
```

Comment:
Curious, was the panic originating in this default clause?

```
default:
		contract.Failf("unexpected builtin type %T", node)
		return nil, nil
```

I think `Failf` would cause a panic so it seems plausible.

We could perhaps consider changing that so that instead of panic-ing we return a placeholder expression like "TODO" and then a non-empty diagnostic. That'd fail more gracefully, WDYT?

Or are we close to covering every `case` here?

Reply:
Yes – `Panic:            fatal: A failure has occurred: unexpected builtin type *ast.SplitExpr`

I think we should have every case covered here, it looks like `Fn::Split` may have been overlooked as it appears to be missing from our docs as well

Reply:
It is. @aq17 could you ensure it's added to our docs? I think @cnunciato and a few other folks have noticed its absence.